composer.json

```diff
@@ -18,9 +18,9 @@
         }
     ],
     "require": {
-      "ext-curl": "*",
-      "lib-curl": "*",
-      "php": ">=5.3.0"
+        "php": ">=5.3.0",
```

Comment:
Why change the require order ? They no longer seem to be sorted by alphabetical order.

Reply:
That’s true, according to https://github.com/composer/composer/blob/c917865fe9f62fd0a7fa6db910597ec4259fc7bf/src/Composer/Json/JsonManipulator.php#L110-L146, requirements are sorted by type first, then alphabetically.

Reply:
If you are interested, take a look at

- https://localheinz.com/blog/2018/01/15/normalizing-composer.json/

🤓

Reply:
Thanks for the links !